### PR TITLE
Rename Email Templates

### PR DESCRIPTION
**Pull request by Ronan-Fernandes**
Rename Email templates to Message Templates

### CODE REVIEW

**Review by pt-tsl — APPROVED** [review]

File: .github/workflows/project-workflow-old-renamed-packages.yml
Comment (nossila) [inline .github/workflows/project-workflow-old-renamed-packages.yml:1]: why is this file necessary? seems like we can just remove `baseapp-email-templates` from both `github-actions.yml` and `publish-actions.yml`

File: baseapp-email-templates/setup.py [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+
 setup(
     name="baseapp-email-templates",
-    packages=find_packages(),
-    version="0.1",
+    description="baseapp-email-templates is now baseapp-message-templates",
     long_description=long_description,
     long_description_content_type="text/markdown",
+    version=VERSION,
+    install_requires=["baseapp-message-templates"],
```
Comment (nossila) [inline baseapp-email-templates/setup.py:16]: the package don't need to require itself, we can remove `install_requires`

File: baseapp-message-templates/README.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
+
+# BaseApp E-mail Templates - Django
```
Comment (nossila) [inline baseapp-message-templates/README.md:2]: Lets change this title as well

**Review by nossila — CHANGES_REQUESTED** [review]

File: .github/workflows/project-workflow-old-renamed-packages.yml
Comment (Ronan-Fernandes) [reply .github/workflows/project-workflow-old-renamed-packages.yml:1]: we still want to do a final publish on this, but now that i'm thinking i agree we don't need to run `github-actions` for `baseapp-email-templates`

File: baseapp-email-templates/setup.py (hunk 1 above)
Comment (Ronan-Fernandes) [reply baseapp-email-templates/setup.py:16]: I think we do want to require this since we want the old package to point to this new package if somene runs `pip install baseapp-email-templates`.

I took this change from this steps https://github.com/simonw/pypi-rename

**Review by pablogadhi — APPROVED** [review]

**Review by nossila — APPROVED** [review]

**Review by nossila — APPROVED** [review]
